src/main/java/org/swisspush/redisques/RedisQues.java

```diff
@@ -883,60 +901,64 @@ private void resetConsumers() {
+                log.warn("Failed to refresh registration for queue '{}'.", queueName, event.cause());
                 // We should return here. See: "https://softwareengineering.stackexchange.com/a/190535"
             }
             // Make sure that I am still the registered consumer
-            String key = getConsumersPrefix() + queue;
+            String consumerKey = consumersPrefix + queueName;
             if (log.isTraceEnabled()) {
-                log.trace("RedisQues consume get: " + key);
+                log.trace("RedisQues consume get: " + consumerKey);
             }
-            redisClient.get(key, event1 -> {
+            redisClient.get(consumerKey, event1 -> {
                 if (event1.failed()) {
-                    log.error("Unable to get consumer for queue " + queue, event1.cause());
+                    log.error("Unable to get consumer for queue " + queueName, event1.cause());
                     return;
                 }
                 String consumer = event1.result();
                 if (log.isTraceEnabled()) {
                     log.trace("RedisQues refresh registration consumer: " + consumer);
                 }
                 if (uid.equals(consumer)) {
-                    QueueState state = myQueues.get(queue);
+                    QueueState state = myQueues.get(queueName);
                     if (log.isTraceEnabled()) {
-                        log.trace("RedisQues consumer: " + consumer + " queue: " + queue + " state: " + state);
+                        log.trace("RedisQues consumer: " + consumer + " queue: " + queueName + " state: " + state);
                     }
                     // Get the next message only once the previous has
                     // been completely processed
                     if (state != QueueState.CONSUMING) {
-                        myQueues.put(queue, QueueState.CONSUMING);
+                        myQueues.put(queueName, QueueState.CONSUMING);
                         if (state == null) {
                             // No previous state was stored. Maybe the
                             // consumer was restarted
-                            log.warn("Received request to consume from a queue I did not know about: " + queue);
+                            log.warn("Received request to consume from a queue I did not know about: " + queueName);
+                        }
+                        if (log.isDebugEnabled()) {
+                            log.debug("RedisQues Starting to consume queue " + queueName);
                         }
-                        log.debug("RedisQues Starting to consume queue " + queue);
-                        readQueue(queue);
+                        readQueue(queueName);
 
                     } else {
-                        log.debug("RedisQues Queue " + queue + " is already beeing consumed");
+                        log.debug("RedisQues Queue " + queueName + " is already being consumed");
```

Comment:
What about
```java
if (log.isDebugEnabled()) {
	//.....
}
```
?

Reply:
I think when using:
```java
log.debug("RedisQues Queue {} is already being consumed", queueName);
```
A loglevel check would be unneccesary anyway.

Reply:
Will do. Thx